frontend/.eslintrc.js

```diff
@@ -403,7 +403,7 @@ module.exports = {
             eventHandlerPropPrefix: 'on'
           }
         ],
-        'react/jsx-no-bind': ['error', { ignoreRefs: true }],
+        'react/jsx-no-bind': 'off',
```

Comment:
This rule is somewhat inefficient with functional component

Respecting it would mean using `useCallback` everywhere

Reply:
You should be using `useCallback` in those instances though. Otherwise every render the function is recreated.